SUMMARY.md

```diff
@@ -124,9 +124,10 @@
   - [Data Providers](libs/application/data-providers/README.md)
   - [Template API Modules](libs/application/template-api-modules/README.md)
   - [Template Loader](libs/application/template-loader/README.md)
-    - [Children Residence Change Template](libs/application/templates/children-residence-change/README.md)
     - [Application Templates Data Protection Complaint](libs/application/templates/data-protection-complaint/README.md)
     - [Document Provider Onboarding](libs/application/templates/document-provider-onboarding/README.md)
+      - [Children Residence Change Template](libs/application/templates/family-matters/children-residence-change/README.md)
```

Comment:
Missing a README at the root of family-matters https://docs.devland.is/repository/gitbook-contribute#root-readme-md-within-handbook-directory